manifests/service.pp

```diff
@@ -58,16 +61,17 @@
 #  }
 #
 define consul::service(
-  $address                  = undef,
-  $checks                   = [],
-  $enable_tag_override      = false,
-  $ensure                   = present,
-  $id                       = $title,
-  $port                     = undef,
-  $service_name             = $title,
-  Hash $service_config_hash = {},
-  $tags                     = [],
-  $token                    = undef,
+  $address                   = undef,
+  $checks                    = [],
+  $enable_tag_override       = false,
+  $ensure                    = present,
+  $id                        = $title,
+  $port                      = undef,
+  $service_name              = $title,
+  Hash $service_config_hash  = {},
+  $tags                      = [],
+  $token                     = undef,
+  Hash[String, String] $meta = {},
```

Comment:
Can you please enforce the minimal string lengt:
```suggestion
  Hash[String[1], String[1]] $meta = {},
```

Please also align all the `=` chars in the parameter section.